user-guide/Advanced_Modules/SRM/SRM_Quarantine.md

```diff
@@ -0,0 +1,438 @@
+---
+uid: SRM_Quarantine
+---
+
+# Quarantine
+
+## About the quarantine state
+
+The quarantine state is a special state of a booking, which indicates that the booking cannot start because one or more resources that were assigned to it are no longer available. Bookings are automatically moved to this state when the Resource Manager detects such a situation.
```

Comment:
Is there only one Resource Manager, or can there be several? If there's only one, we can use "Resource Manager" like a DataMiner module name and leave out "the" in front of it (also below).

Reply:
There is only one, I removed "the" in front of "Resource Manager" everywhere.